src/pages/index.tsx

```diff
@@ -6,6 +6,12 @@ import { GetStartedPopover } from '@/components/GetStartedPopover';
 import { IconChevron } from '@/components/Icons';
 
 import { trackPageVisit } from '@/utils/track';
+import LinkCard from '@/components/LinkCard';
```

Comment:
@hbuchel Should this be in that file that allows components to be used anywhere?

Reply:
I don't see this on the designs for "inner" pages so I think it's ok to just leave out from that for now.